README.md

```diff
@@ -9,11 +9,6 @@ Introduction
   minimum, comply with every ![](badge/req.png) point in this document (except
   where otherwise stated).  For this reason, things that are ![](badge/req.png)
   should not be needlessly specific.
-    - For example, this is a **bad** ![](badge/req.png) point: *A Tox client
```

Comment:
well sure, but i think you get this point across just by saying "overly specific rules are bad", which is what we've done here